torch/csrc/distributed/rpc/message.h

```diff
@@ -115,6 +115,17 @@ class TORCH_API Message final {
   int64_t id_ = -1;
 };
 
+// Create a response Message with an exception for the provided request message.
+// The exception string representation will be used as the message's payload.
+TORCH_API Message
+createExceptionResponse(const Message& request, const std::exception& e);
+
+// Create a response Message with an exception type for the provided request
+// message. The passed in string will be used as the created message's payload
+TORCH_API Message createExceptionResponse(
```

Comment:
It doesn't look like this method is used directly. Is there a reason we have this method? If not, can we remove it and only have one `createExceptionResponse` method for now?

Reply:
I am planning on using them in the 2 PRs I have linked in the description, so I just added it here to remove it from the other 2.